app/constants/forms.js

```diff
@@ -1,9 +1,11 @@
 const constants = {
   errorCodes: {
-    validationError: 400
+    validationError: 400,
+    notFoundError: 404
   },
```

Comment:
Like this 👍 would be good to do in portal too when we next get a chance.

Reply:
Haha - I didn't have a choice as SonarCloud thought it was a critical error to use an explicit 400 or 404 etc without having them in a constant. It says 'No magic number'. Seems to be a new thing that it reports